src/route/lwtunnel.rs

```diff
@@ -106,31 +126,160 @@ impl std::fmt::Display for RouteLwEnCapType {
     }
 }
 
+#[derive(Debug, PartialEq, Eq, Clone, Copy, Default)]
+pub enum RouteIp6Tunnel {
```

Comment:
Please add `Other(DefaultNla)` to hold unknown NLA.

Reply:
Done [2e5ffe9](https://github.com/rust-netlink/netlink-packet-route/pull/148/commits/2e5ffe94d24ce9f5ff4bb147eaa713d989069db5)